include/vctr/PlatformVectorOps/IntelIPP.h

```diff
@@ -110,6 +122,8 @@ class IntelIPP<std::complex<float>>
     static void mul (const std::complex<float>* srcA, std::complex<float> srcB,        std::complex<float>* dst, int len) { assertIppNoErr (ippsMulC_32fc (fc (srcA), fc (srcB), fc (dst), len)); }
     static void div (const std::complex<float>* srcA, const std::complex<float>* srcB, std::complex<float>* dst, int len) { assertIppNoErr (ippsDiv_32fc (fc (srcB), fc (srcA), fc (dst), len)); }
     static void div (const std::complex<float>* srcA, std::complex<float> srcB,        std::complex<float>* dst, int len) { assertIppNoErr (ippsDivC_32fc (fc (srcA), fc (srcB), fc (dst), len)); }
+
+    static std::complex<float> sum (const std::complex<float>* src, int len) { std::complex<float> r; assertIppNoErr (ippsSum_32fc (fc (src), len, fc (&r), ippAlgHintNone)); return r; }
```

Comment:
perhaps `maxAbs` and `minAbs` would also make sense for complex floats?

Reply:
they do from a logical point of view but those versions are not supplied by IPP unfortunately: https://www.intel.com/content/www/us/en/develop/documentation/ipp-dev-reference/top/volume-1-signal-and-data-processing/essential-functions/statistical-functions/maxabs.html

Since this class acts only as a thin abstraction wrapper to what is supplied by the IPP library it wouldn't make to add it here. But the expression implementation itself should work with complex floats